client/README.md

```diff
@@ -55,14 +56,25 @@ api_client = client.Client(API_URL, CREDENTIALS_LOGIN, CREDENTIALS_PASSWORD)
 
 Use it to query alerts:
 ```python
-# Send alerts:
-api_client.send_alert()
 
-# Send medias:
-api_client.send_medias()
+#AS A DEVICE:
+## Create a device
+event_id = client.create_event(lat=10, lon=10).json()["id"]
```

Comment:
Should not it be the instance of the class Client, api_client, to use in the following examples ?
```suggestion
event_id = api_client.create_event(lat=10, lon=10).json()["id"]
```